pom.xml

```diff
@@ -235,6 +215,14 @@
 			<artifactId>joml</artifactId>
 		</dependency>
 
+<!--		only added to resolve maven issues-->
```

Comment:
Which Maven issues does this resolve? Let's try to resolve it in a better way—I'd rather avoid proliferating imagescience dependencies, since it is not an open source project.

Reply:
It was something from Trainable Segmentation, but only happened during copy-jars. We don't depend on Trainable Segmentation, but I guess Trainable Segmentation gets readded during the update site upload. I just needed something to get past the issu ecf, v  vc

Reply:
I'm not sure whether this particular issue was addressed by any of the slew of fixes in scijava-maven-plugin or pom-scijava. Is it still a problem with the new pom-scijava 29.0.0-beta-1, @kephale?